include/functions.h

```diff
@@ -1780,10 +1780,10 @@ void* AudioHeap_AllocDmaMemory(AudioAllocPool* pool, u32 size);
 void* AudioHeap_AllocDmaMemoryZeroed(AudioAllocPool* pool, u32 size);
 void* AudioHeap_AllocZeroed(AudioAllocPool* pool, u32 size);
 void* AudioHeap_Alloc(AudioAllocPool* pool, u32 size);
-void AudioHeap_AllocPoolInit(AudioAllocPool* pool, void* ramAddr, u32 size);
-void AudioHeap_PersistentCacheClear(AudioPersistentCache* persistent);
-void AudioHeap_TemporaryCacheClear(AudioTemporaryCache* temporary);
-void AudioHeap_PopCache(s32 tableType);
+void AudioHeap_InitPool(AudioAllocPool* pool, void* ramAddr, u32 size);
+void AudioHeap_ClearPersistentCache(AudioPersistentCache* persistent);
+void AudioHeap_ClearTemporaryCache(AudioTemporaryCache* temporary);
```

Comment:
btw based on usage (in "init" functions) these two would be more "init" too?
(but having `AudioHeap_InitTemporaryCaches`/`AudioHeap_InitTemporaryCache` would be pretty bad...)

Reply:
Could do:
`AudioHeap_InitTemporaryCacheAll`/`AudioHeap_InitTemporaryCache`
or
`AudioHeap_InitAllTemporaryCache`/`AudioHeap_InitTemporaryCache`
perhaps? Would probably have to change all "plurals" to "All" for consistency

Reply:
`AudioHeap_InitAllTemporaryCaches`/`AudioHeap_InitTemporaryCache` sounds good? I don't think "all" should be added to everything for the sake of consistency

Reply:
I went with 
`AudioHeap_ClearTemporaryCache` -> `AudioHeap_InitTemporaryCache`
`AudioHeap_InitTemporaryCaches` -> `AudioHeap_InitTemporaryPoolsAndCaches`